index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+        integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Full stack software Engineer</title>
+</head>
+
+<body>
+    <section class="background-image">
+        <header>
+            <nav class="nav">
+                <div class="container d-flex justify-content-between">
+                    <a href="#" class="my-logo text-decoration-none text--primary weight--700">My Logo</a>
+                    <i class="fa-solid fa-bars text--primary"></i>
+                </div>
+            </nav>
+
+            <div class="container card d-flex align-items-center justify-content-center">
```

Comment:
- [ OPTIONAL] It would be better if this `div` is turned into a section to make it more semantic :recycle: